vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -236,107 +228,178 @@ An `env.json` file to configure the application URLs and client credentials must
 
 Instructions to export a bundle, including how to configure `env.json`, can be found in the [Export and Publish a Bundle tutorial](../../tutorials/create/pb/export-bundle-from-application.md).
 
-## Profile Management
-To manage and switch between different ent configurations, use `ent profile`. 
 
-To switch between different Entando Applications, even if they are in different clusters, `ent profile` uses Kubernetes contexts, kubeconfig files, and/or custom commands (refer to `ent profile first-use-readme`).
+### Custom Resource Wrapper
+Entando supports a series of `ent ecr` commands to facilitate custom resource operations. 
 
-It is common practice for the vendor tool that connects to a cloud Kubernetes to create a Kubernetes context that must be linked to the profile.
+#### Custom Resource Management
 
-An overview of the current connection and profile information is provided via `ent status`.
+Display the list of bundles associated with the current profile:
+```
+ent ecr list
+```
 
-1. Create and switch to a new profile
+Generate a custom resource and deploy it to the current profile:
 ```
-ent pro new [profileName] [EntandoAppName] [namespace]
+ent ecr deploy
+``` 
 
-This sets the minimal profile data and explains next steps.
+#### Determine Custom Resource Identifiers
+
+The helpers `get-bundle-id` and `get-plugin-id` have been added to `ent ecr` to calculate and display unique identifiers related to custom resources. This provides additional security controls around bundle-specific and microservice plugin resources.
+
+Determine the bundle identifier:
+```
+ent ecr get-bundle-id YOUR-BUNDLE-REPOSITORY-URL
 ```
 
-2. Link the current profile to a Kubernetes context (alias of `ent attach-kubectx`)
+Determine the plugin identifier:
+```
+ent ecr get-plugin-id --auto YOUR-BUNDLE-REPOSITORY-URL
+```
+### Profile Management
+To manage and switch between different ent configurations, use `ent profile`. To switch between different Entando Applications, even if they are in different clusters, `ent profile` uses Kubernetes contexts, kubeconfig files, and/or custom commands (refer to `ent profile first-use-readme`).
```

Comment:
ent configuration >> ent profile configuration?? not sure but ent configuration can refer to how and what version your ent is configured?And here we are talking more about profiles?

Reply:
i think it's implied we're talking about profiles here ?? pretty sure an ent configuration is a profile.

Reply:
ent configuration is confusing. What we're doing here is switching between different profiles which help us connect to different Entando Applications. Maybe just change it to "Entando configurations" to avoid confusion with the ent command.

Reply:
The clause "To switch between different Entando Applications, even if they are in different clusters," can be removed since the first sentence explains it.